### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Ditto.

Preview: https://cyberbotics.com/doc/blog/Webots-2022-b-release?version=blog-post-2022b

### CODE REVIEW

File: docs/blog/Webots-2022-b-release.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+---
+
+## ROS2
+
+With this release we are continuing to develop and improve the support for ROS 2 in Webots.
+New features include:
+
+- Added possibility of importing an URDF robot on the fly
+- Added PointCloud2 support for RangeFinder device
+- Use [CadShape](../reference/cadshape.md) node to represent for visual tags with mesh
```
Comment (BenjaminDeleze) [inline docs/blog/Webots-2022-b-release.md:120]: strange sentence

File: docs/blog/Webots-2022-b-release.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- Added support for humble and rolling
+
+And many other bug fixes.
+
+---
+
+## Extra Goodies
+
+- A very requested feature, R2022b comes with native support for Apple M1 processors on macOS
```
Comment (BenjaminDeleze) [inline docs/blog/Webots-2022-b-release.md:129]: Are we sure it works?

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]

File: docs/blog/Webots-2022-b-release.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+This new version is packed with some new features, improvements and, of course, bug fixes.
+But it doesn't come alone.
+Simultaneously we are releasing [webots.cloud](https://webots.cloud), an open-source webservice to share your simulations online.
+
+Here we are going to present some of the main new features, but for a comprehensive list of changes please refer to the [Change Log](../reference/changelog-r2022.md).
+
+## New Robots
+
+New robot models have been added to the Webots library, the [Crazyflie](../guide/crazyflie.md) quadrotor from [Bitcraze](https://www.bitcraze.io/).
+[Franka Emika](https://www.franka.de/)'s [Panda](../guide/panda.md) robot that comes with the [PandaHand](../guide/panda.md#panda-hand) gripper.
```
Comment (omichel) [inline docs/blog/Webots-2022-b-release.md:18]: This is was not a sentence:
```suggestion
The [Franka Emika](https://www.franka.de/)'s [Panda](../guide/panda.md) robot comes with the [PandaHand](../guide/panda.md#panda-hand) gripper.
```

File: docs/blog/Webots-2022-b-release.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+%end
+
+And finally the "music hall" background was introduced.
+
+---
+
+## webots.cloud
+
+[webots.cloud](https://webots.cloud) is an open-source webservice to share simulations online.
+You can share 3D scenes and animations recorded from a simulation, but also fully interactive simulations.
```
Comment (omichel) [inline docs/blog/Webots-2022-b-release.md:50]:
```suggestion
You can share 3D scenes and animations recorded from simulations, but also fully interactive simulations.
```

File: docs/blog/Webots-2022-b-release.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+---
+
+## Controller Improvements
+
+Several improvements have been introduced in the way controllers communicate with Webots.
+Until now, `<extern>` controllers had to be run in the same machine as the simulation, with R2022b however this is no longer the case and it is now possible for the controller to be run over TCP.
+Using the `WEBOTS_CONTROLLER_URL` variable one can specify the url in the format `<protocol>://<machine_ip>:<webots_port>/<robot_name>` where the supported protocols can be `ipc` for a local connection or `tcp` for a remote one.
+Additionally, the way Webots communicates with controllers has been modified, relying on memory mapped files instead of shared memory, allowing for controllers to be run from a docker.
+The change does not introduce a loss of performance.
```
Comment (omichel) [inline docs/blog/Webots-2022-b-release.md:94]:
```suggestion
This change does not introduce any loss of performance.
```

File: docs/blog/Webots-2022-b-release.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+With this release we are continuing to develop and improve the support for ROS 2 in Webots.
+New features include:
+
+- Added possibility of importing an URDF robot on the fly
+- Added PointCloud2 support for RangeFinder device
+- Use [CadShape](../reference/cadshape.md) node to represent visual properties with mesh
+- Added support for humble and rolling
+
+And many other bug fixes.
```
Comment (omichel) [inline docs/blog/Webots-2022-b-release.md:123]:
```suggestion
And many bug fixes.
```

**Review by omichel — CHANGES_REQUESTED** [review]

**Review by omichel — APPROVED** [review]

**Pull request merged**